### DISCUSSION TITLE
Face control model

### DISCUSSION THREAD

**Discussion by TheLukaDragar** [lllyasviel/ControlNet#222] [Q&A]
I observed that sd generates faces that are a little off. What if we trained a control model to fix this?
Using precise face landmarks. It can be done for poses why not faces too :)

I'm wondering how hard of a problem is this and if there is a way I can make a demo with a relatively small amount of computational resources. Thanks!

**Comment by geroldmeisinger** [comment]
Controlnet 1.1 has support for faces https://github.com/lllyasviel/ControlNet-v1-1-nightly#controlnet-11-openpose
